doc/config_options.txt

```diff
@@ -223,7 +223,7 @@ For example: `i915-GVTg_V5_4`
 <!-- config group device-gpu-mdev-device-conf end -->
 <!-- config group device-gpu-mig-device-conf start -->
 ```{config:option} id device-gpu-mig-device-conf
-:shortdesc: "DRM card ID of the GPU device"
+:shortdesc: "DRM card ID of the GPU device or Container Device Interface (CDI) identifier of the card."
```

Comment:
This is too long for a short description - can you break it up? (For example, have "ID of the GPU device" as short description and then explain the different options in the body text.)

Reply:
For what it's worth, we use "Fully-qualified CDI device name" to mean `{{vendor}}/{{class}}={{name}}`

Reply:
fixed